crates/ironrdp-acceptor/src/connection.rs

```diff
@@ -11,8 +11,8 @@ use super::channel_connection::ChannelConnectionSequence;
 use super::finalization::FinalizationSequence;
 use crate::util::{self, wrap_share_data};
 
-const IO_CHANNEL_ID: u16 = 1003;
-const USER_CHANNEL_ID: u16 = 1002;
+pub const IO_CHANNEL_ID: u16 = 1003;
+pub const USER_CHANNEL_ID: u16 = 1002;
```

Comment:
Suggestion: for exposing these constants `ironrdp-svc` may be a better place.

Rationale: 1) these are static channel IDs and 2) in the future it would be preferable for `ironvnc-acceptor` to not depend on `ironvnc-connector`.

Reply:
Fair enough

Reply:
Then I think this should not be exposed as a public item at all. Instead, add fields in the `AcceptorResult` and use that instead in server (leaving the acceptor decide which ID to use and hiding the details from the server crate, even though in practice it will never change).